### ISSUE TITLE
Doesn't run on Fedora 30

### ISSUE THREAD

**Issue by HenrikRClausen** [jamiemcg/Remarkable#306]
Having read about Remarkable, I was rather keen on getting it for my fresh Fedora 30 setup. However, just like I see for F29, F28 and F27, there is a problem with WebKit. And the solutions provided for F28 don't work anymore, as the files are not available.

The message I get when trying to run Remarkable in the shell is:
[henrik@localhost ~]$ remarkable
Traceback (most recent call last):
  File "/usr/bin/remarkable", line 65, in <module>
    import remarkable
  File "/usr/lib/python3/dist-packages/remarkable/__init__.py", line 30, in <module>
    from remarkable import RemarkableWindow
  File "/usr/lib/python3/dist-packages/remarkable/RemarkableWindow.py", line 27, in <module>
    gi.require_version('WebKit', '3.0')
  File "/usr/lib64/python3.7/site-packages/gi/__init__.py", line 129, in require_version
    raise ValueError('Namespace %s not available' % namespace)
ValueError: Namespace WebKit not available

Any cure?

**Comment by Deadolus** [comment]
You probably have the same problem as https://github.com/jamiemcg/Remarkable/issues/286

**Comment by AlinChindea** [comment]
I am running Fedora 30. When I downloaded the .rpm packaged and used the Software app to install it, it didn't work (did not launch). I've then uninstalled it and intalled it again, this time via the Software app - search for `remarkable`. It installed without issues but it launched with a bit of delay - about 5 seconds after I clicked on Launch. Don't know if this helps but that's what worked for me.
